### PR TITLE
B skip lds

### PR DESCRIPTION
**Pull request by ltqin**
1.b matrix skip lds
2.a matrix using 4 buffer

### CODE REVIEW

**Comment by asroy** [comment]
Do we have performance study on this PR (performance number, thread trace, etc)? We would need that before we decide if we want to merge this PR

@ltqin @zjing14

File: include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdlops_skip_b_lds_v1.hpp [hunk 1]
```diff
@@ -0,0 +1,735 @@
...
+    static constexpr auto I0 = Number<0>{};
+    static constexpr auto I1 = Number<1>{};
+    static constexpr auto I2 = Number<2>{};
+    static constexpr auto I3 = Number<3>{};
+    static constexpr auto I4 = Number<4>{};
+    static constexpr auto I5 = Number<5>{};
+    static constexpr auto I6 = Number<6>{};
+    static constexpr auto I7 = Number<7>{};
+
+    static constexpr auto BaseMultK0 = 8;
```
Comment (shaojiewang) [inline include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdlops_skip_b_lds_v1.hpp:115]: Can ```BaseMultK0``` be a template? 
Also a typo, ```BaseMultiK0``` might be better.

File: include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdlops_skip_b_lds_v1.hpp [hunk 2]
```diff
@@ -0,0 +1,735 @@
...
+
+                    static_for<BaseMultK0 / 2, BaseMultK0, 1>{}([&](auto ii) {
+                        blockwise_gemm.Run(a_thread_buf(Number<ii - BaseMultK0 / 2>{}),
+                                           b_thread_buf(Number<ii>{}),
+                                           c_thread_buf);
+                    });
+                });
+            }
+        }
+
```
Comment (shaojiewang) [inline include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdlops_skip_b_lds_v1.hpp:653]: Could you  please move these lines to a pipeline header?

**Review by shaojiewang — CHANGES_REQUESTED** [review]
1. Could you attach any perf data on MI100/MI200 using fp32/fp16?
2. If the layout is row-col-row, how to transpose data for the skipping lds side?

**Comment by asroy** [comment]
Please add license use the script here:

https://confluence.amd.com/display/MLSE/Development+Guide

**Pull request closed without merging**